low-level/framework/platforms/node/raid/raid.py

```diff
@@ -0,0 +1,81 @@
+# Copyright (c) 2001-2020 Seagate Technology LLC and/or its Affiliates
```

Comment:
low-level/framework/platforms/server/raid/raid.py

Need under server, had this comment earlier ..

Reply:
Done